### DISCUSSION TITLE
Why `fetch` wrappers require `src/hooks.server.js` ?

### DISCUSSION THREAD

**Discussion by KaKi87** [sveltejs/kit#10583] [Q&A]
Hello,

When wrapping SvelteKit `load`'s `fetch` with [`ofetch`](https://github.com/unjs/ofetch) or [`itty-fetcher`](https://github.com/kwhitley/itty-fetcher), the following error appears :

> Error: Failed to get response header "content-type" — it must be included by the `filterSerializedResponseHeaders` option: https://kit.svelte.dev/docs/hooks#server-hooks-handle

At first I thought the issue was caused by the latter, but then I also tried with the former and still was experiencing it, making it a Svelte issue.

And, when I tried the suggested fix :

```js
/** @type {import('@sveltejs/kit').Handle} */
export const handle = ({
    event,
    resolve
}) => resolve(
    event,
    {
        filterSerializedResponseHeaders: name => name === 'content-type'
    }
);
```

It works. But why ? From my point of view, I might as well prepend `// Magic, do not touch` :sweat_smile: 

Thanks

**Comment by terwer** [comment]
I have the same question too.
